.buildkite/x-pack/pipeline.xpack.metricbeat.yml

```diff
@@ -81,6 +81,10 @@ steps:
         key: "mandatory-int-test"
         command: |
           set -euo pipefail
+          # docker-compose problem debugging
+          which docker-compose
+          docker-compose --version
+          pwd
```

Comment:
is this still needed?

Reply:
Sorry, accidentally added it. Will remove.